lib/src/Pos/Node/API.hs

```diff
@@ -547,47 +551,130 @@ instance ToSchema (V1 Version) where
         pure $ NamedSchema (Just "V1Version") $ mempty
             & type_ .~ SwaggerString
 
+
+newtype SecurityParameter = SecurityParameter Int
+    deriving (Show, Eq, Generic, ToJSON, FromJSON, Arbitrary)
+
+instance Buildable SecurityParameter where
+    build (SecurityParameter i) = bprint shown i
+
+instance ToSchema SecurityParameter where
+    declareNamedSchema _ =
+        pure $ NamedSchema (Just "SecurityParameter") $ mempty
+            & type_ .~ SwaggerString
+
+
+instance ToSchema (V1 Core.SlotId) where
+    declareNamedSchema _ =
+        pure $ NamedSchema (Just "Core.SlotId") $ mempty
+            & type_ .~ SwaggerString
+
+deriving instance ToJSON (V1 Core.SlotId)
+deriving instance FromJSON (V1 Core.SlotId)
+
+instance Arbitrary (V1 Core.SlotId) where
+    arbitrary = fmap V1 arbitrary
+
+
+instance ToSchema Byte where
+    declareNamedSchema _ =
+        pure $ NamedSchema (Just "Byte") $ mempty
+            & type_ .~ SwaggerString
+
+instance Buildable Byte where
+    build b = bprint shown b
+
+
+instance Arbitrary (V1 Core.TxFeePolicy) where
+    arbitrary = fmap V1 arbitrary
+
+instance ToSchema (V1 Core.TxFeePolicy) where
+    declareNamedSchema _ =
+        pure $ NamedSchema (Just "Core.TxFeePolicy") $ mempty
+            & type_ .~ SwaggerString
```

Comment:
I see this is represented as:

```json
    "feePolicy": {
      "txSizeLinear": {
        "a": 155381,
        "b": 43.946
      }
    }
```

So, not a string, but rather an object with a few properties ;)